backend/app/usecase/changelog/changelog.go

```diff
@@ -92,6 +93,20 @@ func (p Persist) DeleteChange(id string) error {
 	return p.changeLogRepo.DeleteChange(id)
 }
 
+// UpdateChange updates an existing change with given id in data store.
+func (p Persist) UpdateChange(
+	id string,
+	title string,
+	summaryMarkdown *string,
+) (entity.Change, error) {
+	newChange := entity.Change{
```

Comment:
I wonder whether we should pass `user` into this use case. Every user can view the change log but only the ones with permission can update it. I think this means we need to add checks to authorizer.

Reply:
@byliuyang , yes we should. Indeed, that is going to be my next PR. I separated it out into a new PR as we need to add authorizer to all create, update and delete usecases.